dev-lang/gnucobol/metadata.xml

```diff
@@ -2,8 +2,12 @@
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
 	<maintainer type="person">
-		<email>patrick@gentoo.org</email>
-		<name>Patrick Lauer</name>
```

Comment:
Leave him also here, he's on devaway but might have personal interest towards this package when he returns.